baremetal/metal3machine_manager_test.go

```diff
@@ -280,54 +280,19 @@ func m3mObjectMetaNoAnnotations() *metav1.ObjectMeta {
 
 func machineOwnerRefToMachineSet() *capi.Machine {
```

Comment:
I was a bit surprised that a function named `...ToMachineSet()` returns a `Machine` instead of a `MachineSet`, it sounds like a casting function from the name. Then I looked a bit closer and understood. For more clarity, what do you think about naming it `machineOwnedByMachineSet` ?

Reply:
Agree, fixed as suggested!